docs/tests/common-setup.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+_common_setup() {
+    TEST_BREW_PREFIX="$(brew --prefix)"
+    load "${TEST_BREW_PREFIX}/lib/bats-support/load.bash"
+    load "${TEST_BREW_PREFIX}/lib/bats-assert/load.bash"
+    mkdir -p ~/tmp
+    tmpdir=$(mktemp -d ~/tmp/${PROJNAME}.XXXXXX)
+    export DDEV_NO_INSTRUMENTATION=true
+    mkdir -p ${tmpdir} && cd ${tmpdir} || exit
+    ddev delete -Oy ${PROJNAME:-notset}
+}
+
+_common_teardown() {
+  ddev delete -Oy ${PROJNAME}
+  rm -rf ${tmpdir}/${PROJNAME}
```

Comment:
It removed only `~/tmp/my-backdrop-site.hXV1I6/my-backdrop-site`, but not `~/tmp/my-backdrop-site.hXV1I6`
```suggestion
  rm -rf ${tmpdir}
```